<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenDialog</class>
 <widget class="QDialog" name="GenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>272</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnOk">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>确定(&amp;O)</string>
      </property>
      <property name="icon">
       <iconset resource="other/rc.qrc">
        <normaloff>:/image/btn_ok.png</normaloff>:/image/btn_ok.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>取消(&amp;C)</string>
      </property>
      <property name="icon">
       <iconset resource="other/rc.qrc">
        <normaloff>:/image/btn_close.png</normaloff>:/image/btn_close.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
    <item row="2" column="0">
     <widget class="QLabel" name="lbl_difficulty">
      <property name="font">
       <font>
        <family>STHeiti</family>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_type">
      <property name="font">
       <font>
        <family>STHeiti</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_type">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>加法题目</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>减法题目</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>乘法题目</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>除法题目</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_difficulty">
      <item>
       <property name="text">
        <string>简单</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>一般</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>困难</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="other/rc.qrc"/>
 </resources>
 <connections/>
</ui>
